core/jsAndWasmShared/src/PlatformSpecifics.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2019-2023 JetBrains s.r.o. and contributors.
+ * Use of this source code is governed by the Apache 2.0 License that can be found in the LICENSE.txt file.
+ */
+package kotlinx.datetime
+
+internal expect fun getAvailableZoneIdsSet(): Set<String>
+
+public expect interface InteropInterface
+
+@Retention(AnnotationRetention.BINARY)
+@Target(AnnotationTarget.FILE)
+public expect annotation class JsNonModule()
```

Comment:
Why it is required to typealias JsNonModule? It should be available in common stdlib together with JsModule.

Reply:
`JsNonModule` is JS only.

Reply:
Ok, probably the same thing is with `JsModule` then - it's also Js only and its usage is red in the IDE in commonJs source set.

Reply:
Fixed